physionet-django/console/views.py

```diff
@@ -1403,6 +1403,17 @@ def credential_processing(request):
     final_applications = applications.filter(
         credential_review__status=60).order_by('application_datetime')
 
+    if request.method == 'POST':
+        if 'reset_application' in request.POST:
+            try:
+                application = CredentialApplication.objects.get(slug=request.POST['reset_application'])
+            except CredentialApplication.DoesNotExist:
+                raise Http404()
+            application.credential_review.delete()
+            new_review = CredentialReview.objects.create(application=application)
+            application.save()
```

Comment:
I don't think this `application.save()` is needed because the object is not being modified. I was initially worried that the step might re-save the original review object (leading to a duplicate review object) but I don't think this is the case.